src/guest/vm_builder_qemu.cc

```diff
@@ -204,6 +204,18 @@ static bool SetupHugePages(const std::string &mem_size) {
 }
 
 static int SetAvailableVf(void) {
+    boost::filesystem::path vfio_path("/sys/module/vfio/");
```

Comment:
Could you wrap this into a function? like: `static bool LoadVfioPciModule(void){ ... }`

And also replace the code in `static bool PassthroughOnePciDev(const char *pci_id, PciPassthroughAction action)`: line325~329